app/utils/workflow.js

```diff
@@ -210,8 +210,12 @@ export default class WorkflowUtil {
       };
       // Only push dependencies once (to avoid unnecessary clutter)
       // eslint-disable-next-line prettier/prettier
-      if (!tree.children.some(x => x.name === depEntry.name
-            && x.attributes.assetType === depEntry.attributes.assetType)) {
+      if (
```

Comment:
I'm okay to accept the linter suggestion here, but we should remove the eslint-disable line on 212